internal/initialize/security_test.go

```diff
@@ -47,9 +51,9 @@ func TestRestrictedPodSecurityContext(t *testing.T) {
 	// > operators and developers of security-critical applications, as well as
 	// > lower-trust users.
 	t.Run("Restricted", func(t *testing.T) {
-		if assert.Check(t, psc.RunAsNonRoot != nil) {
-			assert.Assert(t, *psc.RunAsNonRoot == true,
-				"Containers must be required to run as non-root users.")
+		if assert.Check(t, psc.RunAsNonRoot == nil) {
```

Comment:
But if, through some regression, `psc.RunAsNonRoot != nil`, would we catch it?

Reply:
Yes. [assert.Check](https://pkg.go.dev/gotest.tools/v3/assert#hdr-Assert_and_Check) fails the test in that case.